core/controllers/profile_test.py

```diff
@@ -224,6 +224,7 @@ class ProfileLinkTests(test_utils.GenericTestBase):
     USERNAME = 'abc123'
```

Comment:
In fact, I just realized it's worse than that. Most tests call signup(). So, when someone runs the backend test suite, they're making a whole bunch of external calls to Gravatar. However, local tests ought to run locally, and not fail due to e.g. a network connection error.

So you'll probably need to stub urlfetch with a local deterministic service in core/tests/test_utils.py. You could perhaps do something similar to this:

    http://blog.rebeiro.net/2012/03/mocking-appengines-urlfetch-service-in.html

(Also, sorry -- this issue seems to have some hidden depths which I didn't realize at the outset.)